### ISSUE TITLE
HDR Color Support

### ISSUE THREAD

**Issue by ScepticDope** [aseprite/aseprite#2051]
I'd like to request HDR Color support for Aseprite. Like Krita does, except not Windows 10 only but also for Windows 8.1.

See: https://krita.org/en/item/krita-4-2-0-the-first-painting-application-to-bring-hdr-support-to-windows/

**Comment by dacap** [comment]
At the moment I think one possibility would be to choose the Display color profile from Sprite > Properties (this is available in [v1.2.10-beta4](https://community.aseprite.org/t/get-aseprite-beta-version/22)) that is assigned to your HDR monitor:

![Screen Shot 2019-04-12 at 13 03 47](https://user-images.githubusercontent.com/39654/56050915-77ffc480-5d23-11e9-8164-a9810b081bfd.png)

Anyway it's not 10-bit color depth 😭

**Comment by codedcosmos** [comment]
What does aseprite use for it's underlying renderer? What parts would need to be replaced to support different color maps & HDR?

Currently Linux doesn't support HDR outside of stadia, but [work is being done to change that](https://gitlab.freedesktop.org/wayland/weston/-/issues/467). 

Windows however does support HDR so I guess that's where the first implementation of HDR for aseprite will have to be.

Also I believe we will have to support OpenEXR or something? Do .PNG files support HDR?

I might be interested in working on this because I eventually want to add HDR support to my vulkan engine and I since I'm using pixel art for it, aseprite supporting HDR would really be ideal for my workflow.

**Comment by Ai-N3rd** [comment]
This would be amazing for unity devs

**Comment by zadirion** [comment]
yeah as a game dev I really want this support, this way we can design light emitting art from aseprite, would take pixel art to the next level. I think a lot of people aren't asking for this because they don't even realize this is an option, but it's really a very important feature for game developers!

**Comment by codedcosmos** [comment]
Linux is slowly gaining HDR support. Maybe I'll work on this if I find the time.

**Comment by Ketoro** [comment]
linux now supports hdr, maybe this can be added now?

**Comment by philippedcote** [comment]
This issue has been open for 7 years now. Any chances it moves forward? Godot 4.7 will support HDR, it would be a good timing I guess! :)

**Comment by Ketoro** [comment]
godot 4.7 released, now its time for aseprite to support hdr
